src/atsu/common/src/types/Waypoint.ts

```diff
@@ -0,0 +1,11 @@
+export class Waypoint {
```

Comment:
This looks like it could just be an interface.

Reply:
fixed